go_modules/spec/dependabot/go_modules/file_updater_spec.rb

```diff
@@ -104,9 +104,14 @@ module declares its path as: go.etcd.io/bbolt
       before do
         exit_status = double(success?: false)
         allow(Open3).to receive(:capture3).and_call_original
-        allow(Open3).to receive(:capture3).with(anything, "go get -d").and_return(["", stderr, exit_status])
+        allow(Open3).to receive(:capture3).with(anything, "go get -d github.com/etcd-io/bbolt@v1.3.5").and_return(["", stderr, exit_status])
       end
 
+      # This is failing and I'm not sure why...
+      # See also discussion here where it seems the author/reviewers agreed on one solution
+      # but then (confusingly) merged a different one w/o explaining why:
+      # https://github.com/dependabot/dependabot-core/pull/3482#discussion_r612343046
```

Comment:
This PR issue got resolved in https://github.com/dependabot/dependabot-core/pull/3660/commits/5bfdf42fb0ce6655f0fad6f93afdcc82dab01ebe#diff-491ccbbf0ae3cfe37dca8fba172104cc5466631d946daf9639deac37b014692eR44